### PR TITLE
Added LuckyPandas to build page

### PR DESCRIPTION
**Pull request by J3fftw1**
## Description

Added my new plugin to the builds page

## Reason

Hitting off licks in the bando
Black X6, Phantom
White X6, look like a panda
Going out like I'm Montana
Hundred killers, hundred hammers
Black X6, Phantom
White X6, panda
Pockets swole, Danny
Selling bar, candy
Man I'm the macho like Randy

### CODE REVIEW

File: resources/repos.json [hunk 1]
```diff
@@ -1059,7 +1059,23 @@
             }
         }
     },
-
+
```
Comment (TheBusyBiscuit) [inline resources/repos.json]:
```suggestion
```

File: resources/repos.json [hunk 2]
```diff
@@ -1059,7 +1059,23 @@
...
+            "abandoned": false,
+            "createJar": true
+        },
+        "dependencies": {
+            "Minecraft Version(s)": {
+                "1": "1.14+"
+            }
+        }
+    },
+
```
Comment (TheBusyBiscuit) [inline resources/repos.json]:
```suggestion
```

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Comment by J3fftw1** [comment]
Thanks ❤️

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
It's missing a README, otherwise it looks good to go

**Comment by J3fftw1** [comment]
Done

**Review by TheBusyBiscuit — APPROVED** [review]
👍🏻

**Pull request merged**